src/App.tsx

```diff
@@ -1,19 +1,167 @@
 import React from 'react';
 import './App.css';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 import users from './api/users';
+import todos from './api/todos';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
+interface State {
+  todosList: Todo[],
+  todoTitle: string,
+  todoExecutor: string,
+  titleError: string,
+  selectError: string,
+  isValidTitle: boolean,
+  isValidSelect: boolean,
+}
 
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
+class App extends React.Component<{}, State> {
+  state = {
+    todosList: [...todos],
+    todoTitle: '',
+    todoExecutor: '',
+    titleError: '',
+    selectError: '',
+    isValidTitle: false,
+    isValidSelect: false,
+  };
+
+  reloadHandler = (event: React.FormEvent<HTMLFormElement>) => (
+    event.preventDefault()
+  );
```

Comment:
Combine this and `addTodo` method and call them on form submit, not on button click.
Form can also be submitted with pressing `Enter`.